src/pages/home/sidebar/AllSettingsScreen.js

```diff
@@ -16,93 +15,62 @@ import ROUTES from '@src/ROUTES';
 
 function AllSettingsScreen() {
     const styles = useThemeStyles();
-    const {isExecuting, singleExecution} = useSingleExecution();
     const waitForNavigate = useWaitForNavigation();
     const {translate} = useLocalize();
-    const activeRoute = useActiveRoute();
+
     /**
      * Retuns a list of menu items data for "everything" settings
      * @returns {Object} object with translationKey, style and items
      */
-    const menuItemsData = useMemo(
-        () => ({
-            sectionStyle: styles.accountSettingsSectionContainer,
-            sectionTranslationKey: 'initialSettingsPage.account',
-            items: [
-                {
-                    translationKey: 'common.workspaces',
-                    icon: Expensicons.Building,
-                    routeName: ROUTES.SETTINGS_WORKSPACES,
+    const menuItems = useMemo(() => {
+        const baseMenuItems = [
+            {
+                translationKey: 'common.workspaces',
+                icon: Expensicons.Building,
+                action: () => {
+                    waitForNavigate(() => {
+                        Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
+                    })();
                 },
-                {
-                    translationKey: 'allSettingsScreen.subscriptions',
-                    icon: Expensicons.MoneyBag,
-                    action: () => {
-                        Link.openOldDotLink(CONST.ADMIN_POLICIES_URL);
-                    },
-                    shouldShowRightIcon: true,
-                    iconRight: Expensicons.NewWindow,
-                    link: CONST.ADMIN_POLICIES_URL,
+                focused: true,
```

Comment:
This is causing https://github.com/Expensify/App/pull/33280#issuecomment-1881886926